rspec-mode.el

```diff
@@ -155,6 +156,8 @@
 (defcustom rspec-docker-command "docker-compose run"
   "Docker command to run."
   :type 'string
+  :safe '(lambda (symbol value)
```

Comment:
Shouldn't it be a function of one argument, though? Does this work for you?

Reply:
You're right on both changes, however this doesn't seem to be working and I still get the interactive prompt every time the `.dir-locals.el` file is evaluated. Further, I once got the option to `y, n, !` respond, where `!` would save it to my custom script automatically, but now that option seems to have left, and I'm only presented with `y, n`. It doesn't seem related to the safe function, though, because putting it back to `stringp` or just dropping it doesn't change the prompt.

I'm not sure what the expected outcome of using the safe function is. The [Safe-File-Variables](https://www.gnu.org/software/emacs/manual/html_node/emacs/Safe-File-Variables.html#Safe-File-Variables) and [Variable-Definition](https://www.gnu.org/software/emacs/manual/html_node/elisp/Variable-Definitions.html#Variable-Definitions) documentation doesn't really connect the dots for me.

Reply:
I dunno, seems to work fine here. I'm going to push that change, please test.

If that still doesn't work, please post the prompt you are seeing.

Reply:
I had some trouble getting the package installed from github, ended up using something like:
```elisp
(rspec-mode :location (recipe :fetcher github :repo "pezra/rspec-mode" :commit "94a04f148922852b591957b0b3b22de5c4237448"))
```

But it does contain the safe lambda when I pull up the rspec-mode.el file.

The prompt I'm seeing is:
![screen shot 2017-10-31 at 9 46 16 am](https://user-images.githubusercontent.com/328491/32230223-ab749262-be20-11e7-8ee2-2836bd20cefb.png)

I'm using the spacemacs environment (again, emacs noob) so I disabled my ruby layer, which was pulling in the rspec-mode package from MELPA. Seems like the install worked. Still getting the prompt. I don't expect you to figure out what part of spacemacs might be interfering here, but it's gonna take me some more time to cut it down to a minimal configuration, recall the proper keybindings, and reproduce. Thanks for your assistance thus far.

Reply:
If you type `M-x find-function RET rspec-mode RET`, what file does it bring you to? Does its directory's absolute name contain a version?

Reply:
`/Users/carlthuringer/.emacs.d/elpa/rspec-mode-20171025.1806/rspec-mode.el` is what I got after finding rspec-mode. Handy thing to learn about, thank you.

Reply:
It just occurred to me that I could nuke my elpa directory and ensure it re-evaluated the package installs, so I did that:
`/Users/carlthuringer/.emacs.d/elpa/rspec-mode-20171025.1806/rspec-mode.el`

I also cleared out my `safe-local-variable-values` configuration, and got this prompt when re-opening:
![screen shot 2017-11-01 at 3 32 58 pm](https://user-images.githubusercontent.com/328491/32296384-208c11b6-bf1a-11e7-9f5a-d28031b52972.png)

But the `**` rspec-docker-command continues to prompt again after acknowleging with a `!`, which the prompt explains only applies to values that `may not be safe (*)`. Maybe there's no way around this for `-command` variables.

Reply:
I don't get it. It works fine in my Emacs 26 snapshot, and in Emacs 25.2, too.

If I add `(rspec-docker-command . "docker-compose run")` to `.dir-locals.el`, this value is picked up silently. If I introduce a typo in the value, I get a prompt. Everything as expected.

What does `(safe-local-variable-p 'rspec-docker-command "docker-compose run")` evaluate to on your machine?

Reply:
Tried `(safe-local-variable-p 'rspec-docker-command "docker-compose run")` in the `eval-expression` prompt: `nil`

Reply:
Even after you `(require 'rspec-mode)`?

If so, that's odd: `safe-local-variable-p` looks up the `safe-local-variable` property of the variable.

Which should be set by the `(defcustom rspec-docker-command ...` form because it includes the `:safe` keyword.

Reply:
It does work! I wasn't properly loading rspec-mode when testing it. After specifically requiring it in the console it behaves as expected. :)

Reply:
Cool! Then I'm merging this, for the history entry and version bump :)

Reply:
Or, I would merge it, but the PR page indicates a conflict 😕